### ISSUE TITLE
Merging after integration

### ISSUE THREAD

**Issue by GTdevbio** [satijalab/seurat#5890]
Hi 

I have 4 datasets (2 stages-5 and 9 and 2 sexes M and F). The sexes of each stage 5M and 5F are fairly similar and do not need to be integrated. The stages (5M and 9M) however do. So i integrated my early stage of one sex with my late stage of the same sex (5M integrated with 9M) and then I integrated 5F with 9F. When I try to merge the integrated data from each sex 5/9F with 5/9M the previous integration of stages is lost. Is there a way to do this without integrating the sexes? I want to see subtle sex differences and when I integrate them they are lost. 

Thank you

**Comment by yuhanH** [comment]
hi @GTdevbio 
I don't fully understand what assay is lost. Also I am wondering why you don't directly integrate 5M, 9M, 5F, 9F.

**Comment by GTdevbio** [comment]
HI 

I will try to explain better. I have 4 samples stage 5 Male, stage 9 Male, stage 5 Female and stage 5 Female.   I am trying to integrate only the stages (5 and 9) for each sex separately so that I have a data set that is Male (this would have both stage 5 and stage 9 integrated called NEW Male)  and a female dataset (with both stage 5 and stage 9 integrated called NEW female). After I do this step  the UMAP for each sex shows that the data has been batch corrected for stage as all the cell from both stages overlap.  I then want to merge (not integrate) the NEW Male dataset with the NEW female dataset. After I do this the UMAP shows a cluster of cells that are male and female stage 5 and a separate cluster of cells that are male and female stage 9 and there is no longer overlap between the two stages. It appears that any batch correction I did for stage was lost when I merged the sexes together. 


I prefer to merge the two NEW datasets instead of integrate so that  I can maintain the subtle sex differences that are lost if you integrate male and females together.
